pkg/xds/cache_mgr.go

```diff
@@ -267,7 +269,7 @@ func (c *CacheMgr) constructAction(port *networking.Port,
 			}
 		}
 
-		if route.Mirror != nil {
+		if c.validateMirror(route) {
```

Comment:
We should not set RquestMirrorPolicies if route.Mirror is nil.

Reply:
my bad